### PR TITLE
Adding Code-Coverage to the Friendica code-base

### PR DESCRIPTION
**Pull request by nupplaphil**
I just added the suggested lines from https://github.com/friendica/friendica/issues/5568#issuecomment-410644451 and now let's have a look :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Friendica Social Communications Server
 ======================================
+[![codecov](https://codecov.io/gh/friendica/friendica/branch/master/graph/badge.svg)](https://codecov.io/gh/friendica/friendica)
```
Comment (tobiasd) [inline README.md]: For me personally those badges are most uninformative, spammy stuff more annoying then having projects with no README file...

Comment (nupplaphil) [reply README.md]: hm .. for me it says somthing about the maturity of the project and they are short-links too..
Why spammy? I mean OK, there are projects with 20 badges - THAT'S of course spammy ;-)

But I think travis & codecov are something like a minimum of information about the code-base

Comment (tobiasd) [reply README.md]: For me it's the reincarnation of geocities. Just it's not blinking rainbow animated gifs *Collect them all!* Oh there is a new badge, lets include it.

But that should not be something blocking for this PR

**Comment by nupplaphil** [comment]
Shall I remove the badge? As far as I can see @tobiasd don't like it

@mrpetovan and @annando what do you say?

I think nevertheless that 1-2 badges are informative, but I don't want to bother you with it :-)

**Comment by nupplaphil** [comment]
@annando it's a emblem which shows - depending on the result of tests (f.e. codecov or travis-ci) - a status of this repository.
"Travis-CI passed" or "Codecov 50%"

I'm with @tobiasd that it could be spammy if we use a lot of them, but against his sceptism, I think  **just** codecov (and travis) is usefull to show the progress of the repo code-base :-) .. But I'm not dogmatic :)

**Comment by tobiasd** [comment]
> I currently don't understand what this is.

I think it is a fractal generator from the file sizes of a project.

(There homepage seems to be broken without alien JS and other stuff, and I don't care enough to activate it)

**Comment by Rudloff** [comment]
[Code coverage](https://phpunit.de/manual/6.5/en/code-coverage-analysis.html) is not about file size, it is about how much of the code is tested by our tests.

**Comment by MrPetovan** [comment]
I don't have a problem with the badge in general, it's small enough that people who are interested by it will notice and people who aren't won't. I do have a problem with the current code coverage of Friendica and that proudly displaying it on the front page of the project may not be the smartest move. Maybe we should reconsider until we get to 90+% code coverage? It's only public display anyway, it doesn't directly help us reach any code coverage goal.

**Comment by annando** [comment]
@MrPetovan I guess that at 50%+ it would be interesting as well, but exposing the current state really is at least shameful.

**Comment by nupplaphil** [comment]
.. somebody powered off the test-server that's why the test ran into failed ..

**Comment by MrPetovan** [comment]
> Nevertheless I think that codecov can show us where we should add more tests and how good we make progress

I think so too, but for now it should be a dev-only metrics, not a publicly displayed badge.

**Pull request merged**